Security/Core/Authentication/Token/AbstractOAuthToken.php

```diff
@@ -112,6 +112,10 @@ public function __unserialize(array $data): void
         }
     }
 
+    public function copyPersistentDataTo(self $token): void
```

Comment:
I'm not sure, I would go probably opposite with: `copyPersistentDataFrom()`

Reply:
hier is no matter '<--' or '-->'